### DISCUSSION TITLE
pls help me

### DISCUSSION THREAD

**Discussion by bulemars** [magenx/Magento-2-server-installation#112] [General]
Switch to the development mode, and the application crashes and cannot be opened

**Comment by artmouse** [comment]
> Switch to the development mode, and the application crashes and cannot be opened

If such questions arise, it is better to contact a specialist. It will be faster and clearer.
Or look for answers on stackoverflow.

**Comment by magenx** [comment]
if you check logs you will see error or where is the issue

the idea is to move this setup to github actions build and deploy
to completely avoid any manual server operations
